pom.xml

```diff
@@ -403,6 +408,16 @@
                 <artifactId>jaxb-runtime</artifactId>
                 <version>2.3.8</version>
             </dependency>
+            <dependency>
```

Comment:
Kotlin should be updated here: https://github.com/iterate-ch/ds3_java_sdk/blob/develop/pom.xml#L30

Reply:
Managed libs can be removed as sdk version has been updated?